<launch>
    <include file="$(find laikago_tracker)/launch/rs_camera.launch" />
    <node pkg="laikago_real"      name="laikago_ros_to_lcm_server" type="walk_lcm" respawn="true" />
    <node pkg="laikago_tracker"   name="target_detector"           type="target_detector.py" output="screen"/>
    <node pkg="laikago_tracker"   name="track_fsm"           type="track_fsm.py" output="screen"/>
</launch>
